Credify/Credify/Network/Requests+Responses.swift

```diff
@@ -7,6 +7,7 @@
 
 import Alamofire
 import Foundation
+import SwiftUI
```

Comment:
This should be mistake. Please remove this

Reply:
I removed it